src/evergreen/evg_config_utils.rs

```diff
@@ -370,7 +411,7 @@ impl EvgConfigUtils for EvgConfigUtilsImpl {
     ///
     /// Value of given variable in the 'generate resmoke task' vars.
     fn get_gen_task_var<'a>(&self, task: &'a EvgTask, var: &str) -> Option<&'a str> {
-        let generate_func = get_generate_resmoke_func(task);
+        let generate_func = get_func_by_name(task, GENERATE_RESMOKE_TASKS);
```

Comment:
I can probably use my new `get_func_vars_by_name` function from below here. Will try this again once other things are in a better state.